schemas/Quote.json

```diff
@@ -28,7 +28,21 @@
       "description": "Time in seconds between proposed_at and expires_at. Set in quotes from payment systems but not valid in actual transfers",
       "$ref": "NonNegativeDuration.json"
     },
-    "destination_expiry_duration": { "$ref": "NonNegativeDuration.json" }
+    "destination_expiry_duration": { "$ref": "NonNegativeDuration.json" },
+    "additional_info": {
```

Comment:
Changed. The only constraint left is that `additional_info` has to be an object. And indeed, it may or may not exist.